editor/editor_properties.cpp

```diff
@@ -4193,7 +4193,7 @@ struct EditorPropertyRangeHint {
 	bool radians = false;
 };
 
-static EditorPropertyRangeHint _parse_range_hint(PropertyHint p_hint, const String &p_hint_text, double p_default_step) {
+static EditorPropertyRangeHint _parse_range_hint(PropertyHint p_hint, const String &p_hint_text, double p_default_step, bool p_integer) {
```

Comment:
```suggestion
static EditorPropertyRangeHint _parse_range_hint(PropertyHint p_hint, const String &p_hint_text, double p_default_step, bool p_integer = false) {
```
This would reduce amount of changes a lot.

Reply:
See #73361 :)